src/quo2/core.cljs

```diff
@@ -30,7 +30,8 @@
     quo2.components.icon
     quo2.components.info.info-message
     quo2.components.info.information-box
-    quo2.components.input.view
```

Comment:
@ulisesmac, updated the namespace for input component to match the agreed format 👍

Reply:
Looks good! :+1: